### DISCUSSION TITLE
How to add FakeQuantize Node in graph

### DISCUSSION THREAD

**Discussion by shjiyang-intel** [openvinotoolkit/nncf#1707] [Q&A]
Hi, 
May I ask how to add FQ node in the model like POT does? 
Here is my situation, I have a torch model and want to convert it to IR of OpenVINO support with FQ node.
But it seems like there are only round and clamp nodes when I  follow the compress guide in README.md
I can see the FakeQuantize code in this repo, but how can I apply that ? Any supports?
Many Thanks!

**Comment by vshampor** [comment]
Greetings, @ShjiYang!

If you are seeing the FakeQuantize nodes in the .onnx representation of the model exported from Torch, then NNCF has done its job properly. OpenVINO's Model Optimizer may convert these FakeQuantize nodes to clamp/round as they see fit in the IR representation obtained from the ONNX, but the functional result should remain the same.

**Reply by shjiyang-intel** [reply]
Thank you for your kind reply!
Maybe I do not describe the situation clearly. I have a torch model with FQ nodes by applying the config like https://github.com/openvinotoolkit/nncf/blob/b09e061a7f8868c62170aee56a69fa9722519365/docs/compression_algorithms/Quantization.md#example-configuration-files.
But only the input tensor has been quantized, how to quantize weights of convolution?  The weights still are FP32 when I open the model by onnx.
